src/runtime/arch_tinygowasm_malloc.go

```diff
@@ -1,5 +1,5 @@
-//go:build tinygo.wasm && !custommalloc
```

Comment:
custommalloc was added recently and not released yet, if adding `gc.custom` it doesn't seem worth having a separate tag for it.